contents/docs/_snippets/groups-intro.mdx

```diff
@@ -1,4 +1,4 @@
-**Groups** are a powerful feature in PostHog that aggregate events based on entities, such as organizations or companies. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.
+**Groups** aggregate events based on entities, such as organizations or companies. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.
```

Comment:
```suggestion
**Groups** aggregate events based on entities, such as organizations or companies. Key for understanding B2B customers, they enable you to deploy feature flags, analyze insights, and run experiments at a group-level, as opposed to a user-level.
```

Don't love my suggested edit but the idea is to: 1) mention B2B, and 2) be a bit more specific as to what you can do with groups.